app/controllers/tournaments_controller.rb

```diff
@@ -8,14 +8,38 @@ class TournamentsController < ApplicationController
   def index
     all_crosstables_events
     @events = Event.all
+    @tournament = Tournament.new
     @tournaments = policy_scope(Tournament)
   end
 
+  def new
+    authorize @tournament, policy_class: TournamentPolicy
+  end
+
+  def create
+    raise
+    @tournament = Tournament.new(tournament_params)
+    @event = params[:tournament][:event]
+    raise
+    @tournament.event = @event
+    @tournament.location = @event.location
```

Comment:
Lines 25-27 make me realize we don't need those fields in the tournament model. We can just call
```
@tournament.event.location
@tournament.event.date
@tournament.event.rounds
```
whenever we need that info